src/main/java/org/openmainframeproject/tersedecompress/TerseDecompress.java

```diff
@@ -95,15 +99,23 @@ else if (outputFileName == null)
     		printUsageAndExit();
     	}
 
+		System.out.println("Attempting to decompress input file (" + inputFileName +") to output file (" + outputFileName +")");
+
```

Comment:
code below has duplicate lines, which may lead to errors later. more cleaner solution would be to create an object of FileOutputStream, while GZIPOutputStream is overload of such.
```suggestion
var outputStream = null;
		try {
		if (outputFileName.endsWith(".gz")) {
			outputStream = new GZIPOutputStream(new FileOutputStream(outputFileName), 8192, true);
		}
		else {
			outputStream = new FileOutputStream(outputFileName);
		}
		}
			try (TerseDecompresser outputWriter = TerseDecompresser.create(new FileInputStream(inputFileName), outputStream ))
			{	 
				outputWriter.TextFlag = textMode;
				outputWriter.decode();
			}
```
suggested code is just to describe the idea, please test it